src/SmartTransactionsController.ts

```diff
@@ -532,6 +542,12 @@ export default class SmartTransactionsController extends StaticIntervalPollingCo
     >;
 
     Object.entries(data).forEach(([uuid, stxStatus]) => {
+      const transactionHash = stxStatus?.minedHash;
+      this.eventEmitter.emit(`${uuid}:status`, stxStatus);
```

Comment:
Where will this event be used if `eventEmitter` is private?

Reply:
Extension was able to use it. More here about the `events` node module: https://www.npmjs.com/package/events